plugins/ScheduledReports/ReportEmailGenerator/AttachedFileReportEmailGenerator.php

```diff
@@ -47,11 +46,11 @@ protected function configureEmail(Mail $mail, GeneratedReport $report)
         $message = $this->getMessageBody($report);
         $mail->setBodyHtml($message);
 
-        $mail->createAttachment(
+        $mail->addAttachment(
             $report->getContents(),
             $this->attachedFileMimeType,
-            Zend_Mime::DISPOSITION_INLINE,
-            Zend_Mime::ENCODING_BASE64,
+            'inline',
```

Comment:
should we create a constant for this as well @sgiehl ? Could maybe also keep BC by keeping the old method name? Not too important though. Would need to make sure to mention this in developer changelog

Reply:
I'll remove the disposition completely and let the transport class automatically use the best value. Will make it actually less complex.